src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -214,9 +202,11 @@ private void loadConfig(Map<String, Object> config, AmazonCloudWatch cloudWatchC
             rule.awsDimensionSelectRegex = (Map<String,List<String>>)yamlMetricRule.get("aws_dimension_select_regex");
           }
           if (yamlMetricRule.containsKey("aws_statistics")) {
-            rule.awsStatistics = (List<String>)yamlMetricRule.get("aws_statistics");
+            rule.awsStatistics = ((List<String>)yamlMetricRule.get("aws_statistics"))
+                    .stream().map(e -> Statistic.fromValue(e)).collect(Collectors.toList());
```

Comment:
Please use a for loop here, we're not using the newer syntaxes in this repo.

Reply:
Done: https://github.com/prometheus/cloudwatch_exporter/pull/307/commits/24b5f95ce54ced965ec7d2b17b12c311f77e0d22